src/server.jsx

```diff
@@ -23,34 +20,35 @@ const bundles = [
 // description, etc).
 export default locals => {
   let { assets } = locals.webpackStats.compilation;
+  let title = locals.paths[locals.path];
 
   return ReactDOMServer.renderToString(
-    <StaticRouter location={ locals.path } context={{}}>
+    <StaticRouter location={locals.path} context={{}}>
       <html>
         <head>
-          <meta charSet="UTF-8" />
+          <meta charset='utf-8' />
```

Comment:
these should be doublequotes

Reply:
I agree. And in general, we should avoid misc formatting changes like line 26, lines 51 - 53, etc. I understand some of my stylistic choices might be a little different but I'd rather us fix this with something like prettier instead of changing things back and forth due to personal preference. Also, these little things create a lot of noise on PR like this, meaning it's hard to tell what's related to the title population and what are just misc formatting things.

Reply:
@skipjack Yeah, another prettier change, submit a PR to add it to the repo. Sorry for the noise.